### PR TITLE
Add test for BundleBridge

### PR DESCRIPTION
**Pull request by JACKLIAO0**
This pull request adds test for BundleBridge, when running the Jacoco, the test coverage for bundlebridge file has reached 53%.

### CODE REVIEW

**Review by sequencer — CHANGES_REQUESTED** [review]

File: diplomacy/test/src/BundleBridgeSpec.scala [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+      class SinkLazyModule extends LazyModule {
+        val sink = new DemoSink
+        lazy val module = new LazyModuleImp(this) {
+          chisel3.assert(sink.bundle === 4.U)
+        }
+      }
+      class TopLazyModule extends LazyModule {
+        val sourceModule = LazyModule(new SourceLazyModule)
+        val sinkModule = LazyModule(new SinkLazyModule)
+        sinkModule.sink :*= sourceModule.source
```
[the remark this answers is not in the document]
Comment (JACKLIAO0) [reply diplomacy/test/src/BundleBridgeSpec.scala:42]: I want to test  the iBindings of left side  is BIND_STAR   and oBindings the right side is  BIND_QUERY  when sink and source node are connectd with  :*=

File: diplomacy/test/src/BundleBridgeSpec.scala [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+        lazy val module = new LazyModuleImp(this) {
+
+          // test oPorts:
+          // lazy val oPorts: Seq[(Int, InwardNode[DO, UO, BO], Parameters, SourceInfo)] = oDirectPorts.map(oTrace) := source
```
[the remark this answers is not in the document]
Comment (JACKLIAO0) [reply diplomacy/test/src/BundleBridgeSpec.scala:46]: this line is definition of oPorts, to make the results of test more clearer

File: diplomacy/test/src/BundleBridgeSpec.scala [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+          utest.assert(sourceModule.source.oBindings.head._3 == BIND_QUERY)
+          utest.assert(sinkModule.sink.oBindings.isEmpty)
+
+          // test diParams
+          utest.assert(sourceModule.source.diParams.isEmpty)
+          utest.assert(sinkModule.sink.diParams.size == 1)
+
+          // test doParams
+          utest.assert(sourceModule.source.doParams.size == 1)
+          utest.assert(sinkModule.sink.doParams.isEmpty)
```
[the remark this answers is not in the document]
Comment (JACKLIAO0) [reply diplomacy/test/src/BundleBridgeSpec.scala:89]: the case like multiple sink with one source is tested in BundleBridgeNexus : 

NexusLM.broadcastnode :*= sourceModule.source
sinkModule.sink :*= NexusLM.broadcastnode
OthersinkModule.sink := NexusLM.broadcastnode

**Review by sequencer — APPROVED** [review]
LGTM

**Pull request merged**